File: Frontend/src/app/core/models/person.model.ts
export interface CreatePersonRequest {
  firstName: string;
  lastName: string;
  email: string;
  phone: string;
  birthDay: string;
  occupationId: number;
  profile: string;
  sex: string;
}

export interface SavePersonResponse {
  id: number;
  message: string;
}

export interface Occupation {
  id: number;
  name: string;
}

File: Backend/Backend/Models/PersonModel.cs
namespace Backend.Models;

public class PersonModel
{
    public int Id { get; set; }
    public string FirstName { get; set; } = string.Empty;
    public string LastName { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string Phone { get; set; } = string.Empty;
    public string BirthDay { get; set; } = string.Empty;
    public int OccupationId { get; set; }
    public string Profile { get; set; } = string.Empty;
    public string Sex { get; set; } = string.Empty;
}
